index.html

```diff
@@ -49,7 +49,7 @@
     </head>
     <body>
 
-    <h1>♥ МАСЯ ♥ </h1>
+    <h1>♥ МАСЯ и ВАНЬКА ♥ </h1>
```

Comment:
I think the title should be in English, because this is the best language ever!! lol